tests/test_qr_input.py

```diff
@@ -0,0 +1,24 @@
+"""
+Test process.
+"""
+
+from modules import qr_input
+
+
+CAMERA = 1
```

Comment:
Is there a reason why this is 1 and not 0?

Reply:
Yes, so I was testing it on my laptop (macbook) and required to use an index of 1 to access the camera properly. Should I change @Xierumeng

Reply:
According to the OpenCV docs for [`VideoCapture`](https://docs.opencv.org/3.4/d8/dfe/classcv_1_1VideoCapture.html#a5d5f5dacb77bbebdcbfb341e3d4355c1) the integer 0 is passed to open the default camera. On your MacBook was there another camera device connected?

Normally we pass 0 since we want to access the default camera.

Reply:
Yes, change it to 0 after you've tested and confirmed your code is working.